### DISCUSSION TITLE
Capstone 23/04 4week

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/23_1_Capstone#12] [General]
4월 24일 ~ 5월 1일

각자 작업 적어주세요

**Comment by LJI028** [comment]
캐릭터 움직임 수정, 2D 스프라이트 작업
개인 콘텐츠 구현

**Comment by fkdl0048** [comment]
기본 데이터 셋 만들기 (외형, 맵, 상점, 미션 등등) 테이블 작성  

PlayFab매니저 만들기 함수,  클래스 분리

**Reply by fkdl0048** [reply]
화요일에 시험끝나서 수요일에 뽝 할 예정..

**Comment by bbabbungtting** [comment]
Tilemap 제작용 Palette 제작
맵 제작 script 적용

**Comment by aquamagic9** [comment]
Vivox 음성채팅과 채팅 프로토타입 제작

**Comment by ThugMS** [comment]
Photon 환경세팅 완료.
서버 동기화 완료.
